conanfile.py

```diff
@@ -139,5 +139,6 @@ def build(self):
     def package(self):
         packager = files.AutoPackager(self)
         packager.run()
-
+        if self.settings.os in ["Macos", "Linux"]:
```

Comment:
No need to check against OS, Windows won't produce a file called CuraEngine (without extension), so it won't copy anything.